### PR TITLE
4.1.3 update

### PR DESCRIPTION
**Pull request by pvanheus**
Built on the work of @robarthur, this fixes a recipe problem with installation of py2neo cli tool.

### CODE REVIEW

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Comment by xylar** [comment]
It looks like you could probable convert your recipe to [noarch](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#architecture-independent-packages) and save yourself a lot of headache.  Any reason not to do that?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,40 +10,45 @@ source:
...
 
 requirements:
   build:
+    - click >=7.0
+    - colorama
+    - neobolt <1.8,>=1.7.3
+    - neotime <1.8,>=1.7.4
+    - prompt_toolkit <2.1,>=2.0.7
+    - pygments >=2.3.0,<2.4
+    - urllib3 <1.25,>=1.23
```
Comment (xylar) [inline recipe/meta.yaml]: I doubt you need most of these during the build phase.  You probably only need pip and python.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,40 +10,45 @@ source:
...
   run:
     - python
-    - neo4j-python-driver ==1.1.0rc1
-    - urllib3
-
+    - click >=2.0
+    - colorama
+    - neo4j-python-driver >=1.6.2,<1.7
+    - neotime ==1.0.0
+    - prompt_toolkit ==1.0.15
```
Comment (xylar) [inline recipe/meta.yaml]: I don't think you want this version of the package, given what you have above in build.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -10,40 +10,45 @@ source:
...
+    - pygments >=2.0
+    - urllib3 <1.23,>=1.21.1
```
Comment (xylar) [inline recipe/meta.yaml]: This is the line that's actually causing the problem.  There is no python 3.7 build of urllib3 with these restrictions.
```
$ conda create -n test_py2neo python=3.7 "click>=2.0" colorama "neo4j-python-driver>=1.6.2,<1.7" "neotime==1.0.0" "prompt_toolkit==1.0.15" "pygments >=2.0" "urllib3<1.23,>=1.21.1"
Collecting package metadata: done
Solving environment: failed

UnsatisfiableError: The following specifications were found to be in conflict:
  - python=3.7
  - urllib3[version='<1.23,>=1.21.1'] -> python=3.6
Use "conda search <package> --info" to see the dependencies for each package.
```

**Comment by xylar** [comment]
In general, I would recommend relaxing upper-bound (`<`) restrictions on dependencies unless you know there's currently an incompatibility or you expect one with a major version change.  Pinning packages to specific versions means you'll have to constantly update dependencies here on `conda-forge` as the dependencies get updated.

**Comment by xylar** [comment]
I see that you're faithfully copying the package versions from the source, so that does seem like a good practice:
https://github.com/technige/py2neo/blob/py2neo-4.1.3/setup.py#L52-L59

But in this case, I would try relaxing the version of urllib3 and see if you can get a python 3.7 build.

**Comment by xylar** [comment]
I was able to do a successful local build by changing the `requirements` section as follows:
```
requirements:
  host:
    - python
    - pip
  run:
    - python
    - click >=2.0
    - colorama
    - neo4j-python-driver >=1.6.2,<1.7
    - neotime ==1.0.0
    - prompt_toolkit ==1.0.15
    - pygments >=2.0
    - urllib3 >=1.21.1
```

I'd also add:
```
  noarch: python
```
to the build section.  This will save you a lot of needless CI, since it is a pure python package.

**Comment by xylar** [comment]
@conda-forge-admin, rerender

After switching to `noarch`, you should ask conda-smithy to rerender.

**Comment by xylar** [comment]
It looks like these changes are doing the trick!

**Comment by xylar** [comment]
Odd that the rerender didn't do anything...

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
Alright, well, once CI finishes, I'd recommend merging this and then doing another PR with build 1 and hope that the a rerender works correctly to remove all but 1 or 2 of the CI tasks.  I can make a PR for you if you like.

By the way, you might as well close all of the automatic PRs that happened for older versions of the library.

**Comment by xylar** [comment]
Okay, great! Rerender finally went through!  You should be good to merge and to close all those other PRs.

**Pull request merged**